<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>McBeServerManager</class>
 <widget class="QMainWindow" name="McBeServerManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minecraft Bedrock Editon Server Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>Beta 2.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="wn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>测试阶段—Beta 2.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>274</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../z001/ss/dev/minecraft.ico</normaloff>../z001/ss/dev/minecraft.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_st">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>2</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_sc">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>77</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>启动command以对服务器进行更复杂的操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_lv">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>152</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>退出软件</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
